install_idm_server.yml

```diff
@@ -0,0 +1,146 @@
+- hosts: localhost
+  vars:
+    chassis_password: "{{ instackenv_content.nodes[0].pm_password }}"
+
+  tasks:
+    - name: add IdM host to inventory file
+      add_host:
+        name: "{{ idm_host_fqdn }}"
+        groups: "idm"
+        ansible_host: "{{ idm_host_fqdn }}"
+        ansible_user: "root"
+        ansible_ssh_private_key_file: "{{ ansible_ssh_key }}"
+        ansible_python_interpreter: "{{ python_interpreter }}"
+
+    - name: Install RHEL 8.2 on the IdM host
+      block:
+        - name: Build the images for hammercli
```

Comment:
can you please reuse [install_os.yml](https://github.com/redhat-performance/jetpack/blob/master/tasks/install_os.yml) for installing os on idm server